src/odemis/driver/xt_client.py

```diff
@@ -1156,6 +1156,7 @@ def __init__(self, name, role, parent, hfw_nomag, has_detector=False, **kwargs):
             pxs = (fov / self._shape[0],
                    fov / self._shape[0])
             self.pixelSize = model.VigilantAttribute(pxs, unit="m", readonly=True)
+            self.pixelSizeScaled = model.VigilantAttribute(pxs, unit="m", readonly=True)
```

Comment:
No, please, do not add a VA to the driver, just because the GUI would like to display something slightly differently.
I think the simplest is create subclass of SEMStream, with a .pixelSize VA, which listens to ebeam.pixelSize and ebeam.scale.
Alternatively, you could add some code to StreamController to display such label... but it's a little tricky to display it only in this specific case.
Another alternative would be to simply add a label from the tab controller into the stream. We've done something similar in the SPARC2 alignment tab. See the function _addMoIEntries().

Moreover, it should be labelled as "pixel size". That's easy: in gui/conf/data.py, add an entry for this VA and specify the "label".